integrationtesting/fixtures/src/main/java/org/opendcs/fixtures/assertions/TimeSeries.java

```diff
@@ -6,15 +6,24 @@
 import org.junit.jupiter.api.Assertions;
 
 import decodes.tsdb.CTimeSeries;
+import ilex.var.NoConversionException;
 import ilex.var.TimedVariable;
 
 public class TimeSeries
 {
-    public static void assertEquals(CTimeSeries expected, CTimeSeries actual, String message)
+    public static void assertEquals(CTimeSeries expected, CTimeSeries actual, String message) throws Exception
     {
-        assertEquals(expected, actual, .0001, message);
+        assertEquals(expected, actual, .0001, message, null, null);
     }
     public static void assertEquals(CTimeSeries expected, CTimeSeries actual, double delta, String message)
+    {
+        assertEquals(expected, actual, delta, message, null, null);
+    }
+    public static void assertEquals(CTimeSeries expected, CTimeSeries actual, String message, Date start, Date end)
+    {
+        assertEquals(expected, actual, .0001, message, start, end);
+    }
+    public static void assertEquals(CTimeSeries expected, CTimeSeries actual, double delta, String message, Date start, Date end)
```

Comment:
logic  is good. but now that i see it i wonder if a separate "assertContains" might be more appropropriate?